src/main/java/nus/climods/model/module/Module.java

```diff
@@ -265,8 +265,6 @@ public boolean isLessonTypeEnumSelectable(LessonTypeEnum lessonType, SemestersEn
      */
     public HashMap<LessonTypeEnum, ModuleLessonIdMap> getLessons(SemestersEnum semester) {
         requireNonNull(lessonMap);
-        assert lessonMap.containsKey(semester);
```

Comment:
This assertion causes view command to break.
Reason: `ModuleCard::showLessonInformation` checks all semesters in `SemesterEnum` for lessons and most modules do not have lessons in all semesters.